src/golang/lib/job/databricks.go

```diff
@@ -0,0 +1,189 @@
+package job
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
+	databricks_lib "github.com/aqueducthq/aqueduct/lib/databricks"
```

Comment:
nit: Is there a reason for this alias if the sdk package already has an alias, which im assuming is to prevent package import clashes.

Reply:
Both our package and the external one are named `databricks`, so I thought it'd be better to explicitly alias both of them to avoid confusion. I'll keep the `databricks` name for the external library and alias `lib/databricks` to `databricks_lib`.